src/fontra/views/editor/editor.js

```diff
@@ -1780,6 +1845,175 @@ export class EditorController {
     });
   }
 
+  canAddAnchor() {
+    return this.sceneModel.getSelectedPositionedGlyph()?.glyph.canEdit;
+  }
+
+  async doAddAnchor() {
+    const point = this.sceneController.selectedGlyphPoint(this.contextMenuPosition);
+    const { anchor: tempAnchor } = await this.doAddEditAnchorDialog(undefined, point);
+    if (!tempAnchor) {
+      return;
+    }
+
+    let newAnchor = {
+      name: tempAnchor.name ? tempAnchor.name : "anchorName",
+      x: tempAnchor.x ? tempAnchor.x : Math.round(point.x),
+      y: tempAnchor.y ? tempAnchor.y : Math.round(point.y),
+    };
+    const instance = this.sceneModel.getSelectedPositionedGlyph().glyph.instance;
+    let relativeScaleX;
```

Comment:
```js
   const relativeScaleX = layerGlyph.xAdvance ? relativeScaleX = point.x / instance.xAdvance : null;
```

(No need to round here, right?)

Reply:
Right. I changed it based on your suggestion.